### DISCUSSION TITLE
Docker won't start

### DISCUSSION THREAD

**Discussion by carlchen2007** [qdm12/ddns-updater#408] [General]
When I try to start the Docker in Unraid I get the Error in the Logs: 'stat /updater/data/updates.json: not a directory'.
Does smbd has a soloution?

**Comment by qdm12** [comment]
Share your docker run configuration please, this is likely a config error.

**Comment by tgaertner** [comment]
Had the same error.

Go to your docker settings and change 

Config: from "/mnt/user/appdata/ddns-updater" to "/mnt/user/appdata/ddns-updater/data" 

At least it solved it for me.

**Comment by haferm** [comment]
follow exact Setup process: https://github.com/qdm12/ddns-updater#setup
